src/XrdCeph/XrdCephPosix.cc

```diff
@@ -706,8 +707,16 @@ int ceph_posix_open(XrdOucEnv* env, const char *pathname, int flags, mode_t mode
        logwrapper((char*)"Could not find size or stripe_unit xattr for %s", fr.name.c_str());
       }
      else{
-       fr.stripeUnit = std::stoull(d_stripeUnit.c_str());
-       fr.objectSize = std::stoull(d_objectSize.c_str());
+       char cleanStripeUnit[1024];
```

Comment:
Maybe worth adding a comment that `c_str` method of bufferlist does not add NULL-terminator, that's why we have all this hassle.